conda_libmamba_solver/index.py

```diff
@@ -475,6 +475,13 @@ def _package_info_from_package_record(self, record: PackageRecord) -> PackageInf
             noarch = NoArchType(record.noarch.value.title())
         else:
             noarch = NoArchType("No")
+        # include the python_site_packages_path attribute if libmambapy includes support
+        if hasattr(PackageInfo, "python_site_packages_path"):
```

Comment:
With this we are betting that libmamba will use this field name and not something else. Would rather see https://github.com/mamba-org/mamba/pull/3579 merged first 🤔